Makefile

```diff
@@ -38,6 +38,8 @@ fmt: README.md
 README.md: README.md.gotmpl values.yaml
 	@echo "--- Generating documentation"
 	helm-docs --template-files=$<
+	@echo -e "<!-- DO NOT EDIT. THIS IS GENERATED FROM README.md.gotmpl -->\n\n$$(cat README.md)" > README.md
```

Comment:
Love this addition, thanks!

Reply:
It doesn't really matter, but an alternative to doing this is just to add the text to the README.md.gotmpl itself at the top, and since it's a Go template, you can add a comment to explain that the line is for the generated output, like:

```
{{/* Add a comment to the generated output */}}
<!-- DO NOT EDIT. THIS IS GENERATED FROM README.md.gotmpl -->
```

But doing it in the makefile is also fine :man_shrugging: